sparkle/src/components/ScrollArea.tsx

```diff
@@ -0,0 +1,57 @@
+import * as ScrollAreaPrimitive from "@radix-ui/react-scroll-area";
+import * as React from "react";
+import styled from "styled-components";
+
+import { cn } from "@sparkle/lib/utils";
+
+const StyledViewport = styled(ScrollAreaPrimitive.Viewport)`
```

Comment:
Since we don't use styled components, let's continue avoiding them and stick with Tailwind.

Reply:
In this case, it is in order to fix a bug.
ScrollArea adds a display table that prevent to resize in width correctly in some case.

**So 100% not making it a pattern, but would keep this one around with proper commenting**